src/analysis/retail/shaman/elemental/CONFIG.tsx

```diff
@@ -51,7 +52,7 @@ export default {
   ),
   // A recent example report to see interesting parts of the spec. Will be shown on the homepage.
   exampleReport:
-    '/report/YgrNJWQpCwh4XBfj/5-Normal+The+Primal+Council+-+Kill+(4:15)/Blitzlimo/standard',
+    'http://localhost:3000/report/L1R2zVKY8ZDxCTNb/15-Mythic+Scalecommander+Sarkareth+-+Kill+(7:21)/Nerdshockz/standard',
```

Comment:
```suggestion
    '/report/L1R2zVKY8ZDxCTNb/15-Mythic+Scalecommander+Sarkareth+-+Kill+(7:21)/Nerdshockz/standard',
```

omit the domain for this. viewers won't have `localhost:3000` up :)

Reply:
LOL, how did I miss that. :D